research.md

```
@@ -0,0 +1,386 @@
+        + rec.seq.len()
+        + 1 // newline
+        + 2 // + and newline
+        + rec.qual.len()
+        + 1; // newline
+    num_bytes as u64
+}
+```
+
+**Problem**: The code explicitly acknowledges the bug in a comment. FASTQ format allows comments after the `+` separator:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)